ex5.py

```diff
@@ -0,0 +1,33 @@
+###Exercise 5-1###
+import pandas as pd
+wages=pd.read_table("wages.csv",",")
+gen_years=wages.iloc[0:len(wages),0:2]
+gen_years=gen_years.sort_values(by=['gender','yearsExperience'], ascending=[1,1])
```

Comment:
Use drop_duplicates to only keep the unique combinations
unique_genderYears=genderYears.drop_duplicates()